<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemReplaceDialog</class>
 <widget class="QDialog" name="SystemReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Replace System</string>
  </property>
  <widget class="QComboBox" name="comboBoxLocation">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>82</y>
     <width>351</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="replaceAll">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>452</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace all</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replacePrevious">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>451</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace</string>
   </property>
   <property name="icon">
    <iconset theme="go-up"/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxFind">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>12</y>
     <width>391</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="findPrevious">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>417</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="icon">
    <iconset theme="go-up"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>124</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pattern</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxAdvanced">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>192</y>
     <width>461</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Advanced</string>
   </property>
   <widget class="QRadioButton" name="replacePlainText">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>32</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Replace &amp;plain text</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="replaceRegularExpression">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>60</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Replace with &amp;regular expressions</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="replaceSpecialCharachters">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>89</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Replace &amp;with special characters (\n, \r, \t, \0, \u...,\x...)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="matchWholeWord">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>120</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Match whole word only</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="matchCase">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>150</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Match case</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="includeSubdirectories">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>179</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Include Subdirectories</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>50</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace with</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findNext">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>417</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="icon">
    <iconset theme="go-down"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>86</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Location</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectAll">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>418</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replaceNext">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>450</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace</string>
   </property>
   <property name="icon">
    <iconset theme="go-down"/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxPattern">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>121</y>
     <width>391</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>*ext1, *ext2</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxReplace">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>47</y>
     <width>391</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>16</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBrowse">
   <property name="geometry">
    <rect>
     <x>452</x>
     <y>82</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="advancedOptions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>159</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show advanced options</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_statusLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>494</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ready</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="m_progressBar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>492</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
